packages/git-semver-tags/test.js

```diff
@@ -183,12 +187,14 @@ describe('git-semver-tags', function () {
 
   it('should work with tag prefix option', function (done) {
     writeFileSync('test6', '')
-    shell.exec('git add --all && git commit -m"eigth commit"')
-    shell.exec('git tag ms/6.0.0')
-    shell.exec('git add --all && git commit -m"tenth commit"')
-    shell.exec('git tag ms/7.0.0')
-    shell.exec('git add --all && git commit -m"eleventh commit"')
-    shell.exec('git tag notms/7.0.0')
+    exec('git add --all && git commit -m"eigth commit"')
+    exec('git tag ms/6.0.0')
+    writeFileSync('test6', '1')
+    exec('git add --all && git commit -m"tenth commit"')
+    exec('git tag ms/7.0.0')
+    writeFileSync('test6', '2')
+    exec('git add --all && git commit -m"eleventh commit"')
```

Comment:
before git add command silently failed and where not executed as there was no changes to be committed
now, command is going to throw error if there is nothing to commit